ONSdigital/response-operations-ui
# update design system to 62.2.4

# What and why?
This PR updates the design system version of ROPS to 62.2.4 to being it to the most recent version before breaking changes in version 63.0.0
# How to test?
Deploy Rops and ensure everything is working as normal, there were breaking changes in 62.0.0 involving the breadcrumbs so check they are working correctly on each of the pages. 
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&view=detail&selectedIssue=RAS-817
